packages/terra-clinical-result/src/ClinicalResultBloodPressure.jsx

```diff
@@ -6,7 +6,10 @@ import ThemeContext from 'terra-theme-context';
 import IconModified from 'terra-icon/lib/icon/IconModified';
 import IconComment from 'terra-icon/lib/icon/IconComment';
 import IconUnverified from 'terra-icon/lib/icon/IconDiamond';
+import VisuallyHiddenText from 'terra-visually-hidden-text';
 import { injectIntl } from 'react-intl';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import uuidv4 from 'uuid/v4';
```

Comment:
This needs to be updated to use the import from uuid@8 once that is pulled in.

Reply:
Updated here - https://github.com/cerner/terra-clinical/pull/868/commits/f932e12eaad49881c42630811d1619d1000807ed#diff-42afe5315e2b63b43f755f1ec567ef32b1ad903109d718db4e7526f825680f6aR11